akd_local_auditor/src/storage/s3/test.rs

```diff
@@ -308,14 +245,69 @@ async fn integration_test_audit_verification() -> Result<()> {
     Ok(())
 }
 
+// ============================ IMPORTANT ============================ //
+// NEVER run this operation in a CI pipeline, it takes 40+ min to complete
+// on reasonable hardware, and is only used for manual testing & timing information
+#[tokio::test]
+#[ignore]
+async fn expensive_audit_verification() -> Result<()> {
```

Comment:
Maybe add a non-expensive audit verification to test this functionality works?

Reply:
added